uwrt_arm/uwrt_arm_controllers/uwrt_cartesian_controller/uwrt_cartesian_controller_plugins.xml

```diff
@@ -0,0 +1,9 @@
+<library path="lib/libcartesian_controller">
+
+  <class name="velocity_controllers/UWRTCartesianController" type="velocity_controllers::UWRTCartesianController" base_class_type="controller_interface::ControllerBase">
+    <description>
+      UWRTCartesianController which accepts TwistStamped Msg.
```

Comment:
What does it do? Accepts an End Effector Twist Msg converts it into individual Joint Velocity Cmds